students/smitco/lesson02/series.py

```diff
@@ -0,0 +1,48 @@
+#lesson 02 series 
+#step 1- print fibonacci value
+def fibonacci(n):
+    fib = []
+    for num in range(n+1):
+        if num == 0:
+            fib.append(0)
+        elif num == 1:
+            fib.append(1)
+        else:
+            f = fib[num-1] + fib[num-2]
+            fib.append(f)
+    return(fib[n])   
```

Comment:
Excellent!  Though the assignment technically asks to 'return' the value, a print statement makes sense here to see the response.